README.md

```diff
@@ -19,6 +19,19 @@ This dataset includes Direct Borohydride Fuel Cell (DBFC) impedance and polariza
 
 We have provided some **Jupyter Notebooks** to visualize the data, visit [here](https://github.com/ECSIM/dbfc-dataset/tree/master/Notebooks)
 
+
+## Issues & Bug Reports			
+
+Just fill an issue and describe it. We'll check it ASAP!							
+or send an email to [data@ecsim.ir](mailto:data@ecsim.ir "data@ecsim.ir"). 
+
+You can also join our discord server			
+
+<a href="https://discord.gg/ZwrNZYVM4G">
```

Comment:
Use this link :  https://discord.gg/NDwgZadaDH

Reply:
Sure